### DISCUSSION TITLE
File Flagged by Anti-Virus

### DISCUSSION THREAD

**Discussion by XcomNutt** [X2CommunityCore/xcom2-launcher#325] [General]
Tried to download and install the alternative launch file, though the application itself was Flagged and subsequently deleted by my anti-virus. please be certain that your files do not have any such virus.

**Comment by RevZero** [comment]
You can be pretty sure, that the original ZIP is clean. I just ran it through an online scanner and one tool detected a "false positive". What Antivir are you using?

![2022-06-24 12 24 29 www virustotal com 9c030037eb29](https://user-images.githubusercontent.com/936992/175516683-7fae2d17-62d5-45e3-b01b-b7aa24886a2a.png)
